src/components/container/createContainer.tsx

```diff
@@ -1,56 +1,46 @@
 import * as React from 'react'
 import {useEffect, useRef, useState} from "react";
-import {type_alias} from "../customTypings";
+import {useDispatch} from "react-redux";
+import {pushTasks, deleteTask} from "../redux/todoSlice";
+import {TaskStatus} from "../customTypings";
 
 export type PropsCreateContainer = {
     text: string,
-    id: number,
-    tasks: type_alias[],
-    task: type_alias,
-    deleteTask:  React.Dispatch<type_alias[]>,
-    todos: type_alias[],
-    setTodos:  React.Dispatch<type_alias[]>,
-    durings: type_alias[],
-    setDurings:  React.Dispatch<type_alias[]>,
-    dones: type_alias[],
-    setDones:  React.Dispatch<type_alias[]>,
-    taskStatus: string,
+    id: string,
+    taskStatus: TaskStatus,
 }
 
-export default function CreateContainer ({text,id,tasks,task,deleteTask,todos,setTodos,durings,setDurings,dones,setDones,taskStatus}:PropsCreateContainer) {
-
+export default function CreateContainer ({text,id,taskStatus}:PropsCreateContainer) {
+    const dispatch = useDispatch();
     const [options, setOptions] = useState<boolean>(false)

```

Comment:
`<boolean>` rzutowanie niepotrzebne bo widać jasno z stanu początkowego ze stan trzyma `boolean`.

Reply:
`options` brzmi jak tablica jakiś opcji a jest to boolean. Przyda się bardziej zrozumiała nazwa